README.md

```diff
@@ -498,6 +498,99 @@ Flag | Notes
 `--https_proxy`      | Needed for package manager or Ubuntu guest and for from github downloaded packages
 `--ftp_proxy`       | Needed for some Perfkit packages
 
+CONFIGURATIONS AND CONFIGURATION OVERRIDES
+==================
+Each benchmark now has an independent configuration which is written in YAML.
+A user config is a way to override this default configuration on a per benchmark
```

Comment:
Nit: user config isn't used in the title anymore.
Maybe "users may override this default configuration by providing a configuration..."